cassandane/Cassandane/Tiny.pm

```diff
@@ -0,0 +1,10 @@
+package Cassandane::Tiny;
+use strict;
+use warnings;
+
+sub import {
+  $Cassandane::Tiny::Loader::RELOADED = 1;
```

Comment:
This could use a `no warnings 'once';` to shush the "possible typo" warning

Reply:
Done.